packages/usa-in-page-navigation/src/test/in-page-navigation.spec.js

```diff
@@ -0,0 +1,88 @@
+const assert = require("assert");
+const fs = require("fs");
+const path = require("path");
+const sinon = require("sinon");
+const behavior = require("../index");
+
+const HIDE_MAX_WIDTH = 639;
+const TEMPLATE = fs.readFileSync(path.join(__dirname, "/template.html"));
+const STYLES = fs.readFileSync(
+  `${__dirname}/../../../../dist/css/uswds.min.css`
+);
+const THE_NAV = ".usa-in-page-nav";
+const PRIMARY_CONTENT_SELECTOR =
+  ".usa-in-page-nav-container .usa-in-page-nav .usa-in-page-nav-list";
```

Comment:
Updated per the BEM class update described [here](https://github.com/uswds/uswds/pull/4918#issuecomment-1301502261).

```suggestion
  ".usa-in-page-nav-container .usa-in-page-nav .usa-in-page-nav__list";
```